src/ansys/dynamicreporting/core/utils/report_remote_server.py

```diff
@@ -1046,19 +1046,17 @@ def _build_template_data(guid, templates_data, templates, template_guid_id_map):
     for template in templates:
         if template.guid == guid:
             curr_template = template
-    if curr_template is None:
-        return
 
+    fields = ["name", "report_type", "date", "tags", "item_filter"]
```

Comment:
@viseshrp As I only need a subset of the attributes, I still hardcode over here but a bit better than previous way.